server/views/components/image/index.njk

```diff
@@ -9,4 +10,7 @@ data-srcset="
 {% if data.sizesQueries %}
   sizes="{{ data.sizesQueries }}"
 {% endif %}
-alt="{{ model.caption | striptags() if model.caption }}" />
+{% if model.description %}
+  data-description="{{ model.description }}"
+{% endif %}
+alt="{% if model.alt %} {{ model.alt | striptags() }}{% elif model.caption %} {{ model.caption | striptags() }}{% endif %}" />
```

Comment:
I would say that description makes sense.
Caption is something that normally creates a relationship between the content that the image is in, and the image.

Description is something that is tightly coupled to the image that describes it despite it's context?